frontend/app/protected/template.hbs

```diff
@@ -1 +1 @@
-{{liquid-outlet}}
+{{outlet}}
```

Comment:
It's already in the application template file:
https://github.com/adfinis-sygroup/adsycc/blob/master/frontend/app/application/template.hbs